File: Python/questão 1.py
vp = int(input("Digite a velocidade maxima da avenidade: "))
vm = int(input("Digite a velocidade do motorista: "))
if vp < vm:
    multa = (vm - vp) * 5 
    print(f'A multa é de: {multa} reais') 
elif vm <= vp:
    print("Não há multa")

File: C/questão 1.c
#include <stdio.h>
#include <stdlib.h>

void main(){
    int vp, vm, multa;
    int taxa = 5;

    printf("Digite a velocidade permitida: ");
    scanf("%d", &vp);
    printf("Digite a velocidade do motorista: ");
    scanf("%d", &vm);
    multa = (vm - vp) * taxa;
    if(vm > vp){
        printf("Você recebeu uma multa de: %d reais", multa);

    }else{
        printf("Não há multa!");
    }
}
